src/OffsetProgressiveCurve.sol

```diff
@@ -70,14 +70,14 @@ contract OffsetProgressiveCurve is BaseCurve {
         require(slope18 > 0, "PC: Slope must be > 0");
 
         SLOPE = UD60x18.wrap(slope18);
-        HALF_SLOPE = SLOPE.div(UD60x18.wrap(2));
```

Comment:
3.1 - Wrong Use of PRB-Math Library in Progressive Curves
This was also fixed to properly compute the half slope based on the given slope.  This affected the effect of the constructor arguments on the slope.